From 0fbd8684578ee5366484678b9b090d00a2030416 Mon Sep 17 00:00:00 2001
From: Lijun Pan <Lijun.Pan@freescale.com>
Date: Mon, 8 Feb 2016 17:40:13 -0600
Subject: [PATCH 09/12] staging: fsl-mc: dprc device's sysfs rescan
 documentation
Patch-mainline: Queued in subsystem maintainer repository
Git-repo: https://github.com/qoriq-open-source/linux.git nxp/linux-v4.5
Git-commit: 0fbd8684578ee5366484678b9b090d00a2030416
References: fate#319900

Add the sysfs documentation for the rescan attribute of
dprc devices.

Signed-off-by: Lijun Pan <Lijun.Pan@freescale.com>
Signed-off-by: Matthias Brugger <mbrugger@suse.com>
---
 Documentation/ABI/testing/sysfs-devices-dprc | 13 +++++++++++++
 1 file changed, 13 insertions(+)
 create mode 100644 Documentation/ABI/testing/sysfs-devices-dprc

diff --git a/Documentation/ABI/testing/sysfs-devices-dprc b/Documentation/ABI/testing/sysfs-devices-dprc
new file mode 100644
index 0000000..3c5d725
--- /dev/null
+++ b/Documentation/ABI/testing/sysfs-devices-dprc
@@ -0,0 +1,13 @@
+What:		/sys/devices/.../dprc.X/rescan
+Date:		October 2015
+Contact:	Lijun Pan <Lijun.Pan@freescale.com>
+Description:
+		Writing a non-zero value to this attribute will
+		force a rescan of dprc.X container in the system, and
+		add/remove previously added/removed objects in
+		Management Complex firmwarer, i.e., synchronize
+		the objects under dprc.X and Management Complex
+		firmware. For example:
+		# echo 1 > /sys/devices/platform/80c000000.fsl-mc/dprc.1/rescan
+
+
-- 
2.6.6

